src/platforms/common/enriching-events/scopes.mdx

```diff
@@ -40,11 +40,11 @@ routes or controllers.
 ## How the Scope and Hub Work
 
 As you start using an SDK, a scope and hub are automatically created for you out
-of the box. You are unlikely to be interacting with the hub directly unless you
+of the box. It is unlikely that you will interact with the hub directly unless you
```

Comment:
We use contractions wherever we can, but otherwise, this is a nice update.

```suggestion
of the box. It's unlikely that you'll interact with the hub directly unless you're
```

Reply:
Thanks! Haha, I think this was one of the reasons I started editing the docs.

Got it. Yeah, your style is a bit less formal than mine.

Reply:
> Thanks! Haha, I think this was one of the reasons I started editing the docs.
> 
> Got it. Yeah, your style is a bit less formal than mine.

Yeah, we love a contraction around here 😄 but it's just as common to not use them. Thanks for all your work on this!